src/OpenConext/ProfileBundle/Resources/views/MyServices/overview.html.twig

```diff
@@ -76,7 +76,13 @@
                                     {% endif %}
                                     <tr>
                                         <td title="{{ attributeUrn }}">
-                                            <strong>{{ ('profile.saml.attributes.' ~ attribute.attributeDefinition.name)|trans({}, 'saml') }}</strong>
+                                            {% set translatedAttribute = ('profile.saml.attributes.' ~ attribute.attributeDefinition.name)|trans({}, 'saml') %}
+
+                                            {% if translatedAttribute == 'profile.saml.attributes.' ~ attribute.attributeDefinition.name %}
```

Comment:
I think we wanted to make them look like translation keys in order to distinguish the translation key from the raw URN.
I might rename this to `translatedAttributeName`.